plugins/view/src/index.ts

```diff
@@ -328,6 +328,9 @@ export interface Action<T extends Doc = Doc, P = Record<string, any>> extends Do
   // Action is applicable only for objects matching criteria
   query?: DocumentQuery<T>
 
+  // Action is shown only if the check is passed
+  checkIsVisible?: Resource<() => Promise<boolean>>
```

Comment:
I do not see a usable scenario without passing a document(s) planed to be shown. Let's have it named, 
visibilityTester?: Resource<(doc: Doc|Docs[]|undefined) => Promise<boolean>>

Reply:
The scenario is to test the current account, anyway changed the name and arg type for future purposes